javascript-local-storage/data.js

```diff
@@ -0,0 +1,17 @@
+/* exported todos */
+
+let todos = [];
+const previousTodosJSON = localStorage.getItem('javascript-local-storage');
+
+if (previousTodosJSON !== null) {
+  todos = JSON.parse(previousTodosJSON);
+}
+
+const $window = document.defaultView;
```

Comment:
This isn't needed. You can just use `window` directly since it's a global object